invenio_rest/config.py

```diff
@@ -75,3 +75,54 @@
           'marc21': 'application/marcxml+xml'
        }
 """
+
+REST_CSRF_ENABLED = True
+"""Enable CSRF middleware. (Default: ``True``).
+
+.. note::
+   The CSRF middleware accepts some configuration parameters that are not set
+   by default as if they are set they have an impact in the csrf validation
+   workflow. The available options are:
+
+   \`CSRF_METHODS\`: HTTP methods against which the csrf check should run.
+   Defaults to \`['POST', 'PUT', 'PATCH', 'DELETE']\`.
+
+   \`CSRF_HEADER\`: The name of the request header used for CSRF
+   authentication. Defaults to \`X-CSRF-Token\`.
+
+   \`CSRF_COOKIE_NAME\`: The name of the request cookie used for CSRF
+   authentication. Defaults to \`_csrftoken\`.
+
+   \`CSRF_COOKIE_MAX_AGE\`: The maximum time until the cookie expires. After
+   the expiration the cookie will be removed. Defaults to \`60*60*24*7*52\`
+   (1 year).
+
+   \`CSRF_COOKIE_DOMAIN\`: The for which the csrf cookie should be valid.
+   Defaults to \`flask.sessions.SessionInterface.get_cookie_domain\`.
+
+   \`CSRF_COOKIE_PATH\`: The path that should be set for the csrf cookie.
```

Comment:
Not sure I understand this `path`. Can you explain better or provide a link to read more?

Reply:
It's the url path for which the cookie is valid. For example, if path set `/example` the returned cookie would be set only in the `/example` url. I amend the comment.